internal/checker/utilities.go

```diff
@@ -1218,8 +1218,33 @@ func isValidNumberString(s string, roundTripOnly bool) bool {
 	return !n.IsNaN() && !n.IsInf() && (!roundTripOnly || n.String() == s)
 }
 
-func isValidBigIntString(s string, roundTripOnly bool) bool {
-	return false // !!!
+func isValidBigIntString(s string, roundtrip bool) bool {
+	if s == "" {
+		return false
+	}
+	scan := scanner.NewScanner()
```

Comment:
The original code had `skipTrivia=false`, so that needs to also be done here.

Reply:
have updated this as well and have checked that the following tests pass: 

specific: 
go test github.com/microsoft/typescript-go/internal/testrunner -run "TestLocal/bigintTemplateLiteral.ts" 
go test github.com/microsoft/typescript-go/internal/checker

all: 
go test ./...